.devcontainer/devcontainer.json

```diff
@@ -1,18 +1,84 @@
 {
-	"name": "pwndbg",
-	"build": {
-		"context": "..",
-		"dockerfile": "../Dockerfile"
-	},
-	"runArgs": [
-		"--security-opt=no-new-privileges:true",
-		"--cap-drop=ALL",
-		"--cap-add=SYS_PTRACE"
-	],
```

Comment:
This was changed to use `--privileged`: why? The opts used here were more safe for the users